Material.Blazor/Components/DatePicker/InternalDatePickerPanel.razor

```diff
@@ -2,8 +2,7 @@
 
 @inherits InputComponent<DateTime>
 
-
-<div @attributes="@AttributesToSplat()">
+<div class="mdc-typography--body2 mb-dp-container">
```

Comment:
We are the only ones using this component, therefore we know that there aren't attributes to be splat, except those added to ClassMapper and StyleMapper. In this case, only class mapper is relevant, which has a constant value.